content/challenges/medium/8 - Is Isogram/func.js

```diff
@@ -12,5 +12,13 @@
  * @returns {boolean}
  */
 module.exports = (str) => {
-    // your code here
+    let usedLetters = []
+    for (i = 0; i < str.length; i++) {
+        if (usedLetters.includes(str[i].toLowerCase())) {
+            return false
+        } else {
+            usedLetters.push(str[i].toLowerCase())
+        }
+    }
+    return true
```

Comment:
I'm challenging you to use `new Set( something here )` to do this in 3 lines. Look up what Set does and see if you can figure it out.